README.md

```diff
@@ -1 +1,107 @@
+- When logged in on the My Builds page and clicking on the users own builds, 'Edit' will appear at the bottom.
+- By clicking on the top right of the website, or on the hamburger if on mobile, the user can select 'Create a Build' in-order to create a new build.
+  - When creating a new build the user will be able to add any parts they wish from the dropdown provided and populate the fields that appear with the information required by the site (Create). Any of this information can be edited in the future as described previously.
+- On the view page users will be able to vote on other peoples builds.
+
+### Features left to implement
+
+- An option to say whether the item has been brought or not so it can be removed from the running total.
+
+- API in-order to populate the car fields (make, model, trim, year).
+
+## Technologies Used
+
+Below are the libraries and languages used in creating this project.
+
+- [HTML5](https://en.wikipedia.org/wiki/HTML5)
+
+- [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS33)
+
+- [JavaScript](https://www.javascript.com/)
+
+- [jQuery](https://jquery.com/)
+
+- [Font Awesome](https://fontawesome.com)
+
+- [Flask](https://flask.palletsprojects.com/en/1.0.x/)
+  - [Flask-Login](https://flask-login.readthedocs.io/en/latest/)
+
+- [Bcrypt](https://en.wikipedia.org/wiki/Bcrypt)
+
+- [PyMongo](https://api.mongodb.com/python/current/)
+
+## Testing
+
+Auotmated testing can be found in testing.py. It can be ran with `python testing.py` in the appropriate directory on the command line.
+
+## Deployment
+
+This application was deloyed using Heroku. It can be view [here.](http://autolog-msped.herokuapp.com/)
+
+For this application to be deployed it first requires two files, requirements.txt and a Procfile.
+
+Requirements.txt can be created using `pip freeze > requirements.txt` in the command line. The Procfile can be added by typing `echo web: python app.py > Procfile` into the command line.
+
+With both of these file create a new app on [Heroku](https://dashboard.heroku.com/login). Download the Heroku CLI toolbelt and login to Heroku via the command line. Then in the command line type git init, git add ., git commit, git push heroku master. Finally initalise the app on the Heroku server by typing `heroku ps:scale web=1` and press enter.
+
+Now in the Heroku Settings add the CONFIG VARS IP (0.0.0.0), PORT (5000), MONGO_DBNAME and MONGO_URI.
+
+### Local Deployment
+
+To run the site locally, first download the .zip from GitHub and extract it.
+
+In app.py change:
+
+```python
+    if __name__ == '__main__':
+        app.run(host=os.environ.get('IP'),
+                port=int(os.environ.get('PORT')),
+                debug=False)
+```
+
+to
+
+```python
+    if __name__ == '__main__':
+        app.run(debug=False)
+```
+
+This will ensure that the project runs diectly on 127.0.0.0:5000, or if you have specified an IP and PORT on your local systems environment variables already you can use the preivous option.
+
```

Comment:
Include information on creating a virtual environment and activation it, then running pip install -r requirements.txt